src/wasmtime/crates/lind-common/src/lib.rs

```diff
@@ -218,6 +218,33 @@ pub fn add_to_linker<
             wasmtime_lind_multi_process::signal::signal_handler(&mut caller);
         },
     )?;
+    
+    // a temporary solution to have libc_assert_fail correctly working
+    linker.func_wrap(
```

Comment:
these are both basically write() syscalls, instead of adding these host functions can't we just use write? Once the addr translation PR is in that should be easy right? @qianxichen233 @drapl0n